Unbox.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Unbox"
-  s.version      = "1.5.1"
+  s.version      = "1.5.2"
```

Comment:
This should be done after creating a release tag, I can take care of it once the release has been published :)